nvm.sh

```diff
@@ -2441,9 +2471,12 @@ nvm() {
         FLAVOR="$(nvm_node_prefix)"
       fi
 
-      if nvm_is_version_installed "$VERSION"; then
+      if nvm_is_version_installed "$VERSION" && nvm use "$VERSION"; then
         nvm_err "$VERSION is already installed."
```

Comment:
it's very intentional that the error message prints only if `nvm_is_version_installed` fails, and not if `nvm use` fails.